### ISSUE TITLE
Upgrading to httr 1.1.0 causes every network service to fail

### ISSUE THREAD

**Issue by sbpcs59** [DataONEorg/rdataone#125]
httr 1.1.0 (published to cran on 2016-01-28) switched to xml2 (was XML), so httr::content(as="parsed") returns a different data type that our code doesn't handle (now "xml_document", was "XMLInternalDocument"). Here is a sample traceback:

```
-----------
no applicable method for 'xpathApply' applied to an object of class "c('xml_document', 'xml_node')"
1: withCallingHandlers(eval(code, new_test_environment), error = capture_calls, message = function(c) invokeRestart("muffleMessage"))
2: eval(code, new_test_environment)
3: eval(expr, envir, enclos)
4: CNode("STAGING") at test.AuthenticationManager.R:9
5: CNode("STAGING") at /Users/slaughter/git/rdataone/dataone/R/CNode.R:69
6: .local(env, ...)
7: getNodeSet(content(response, as = "parsed"), "/*[local-name() = 'node']") at /Users/slaughter/git/rdataone/dataone/R/CNode.R:118
8: xpathApply(doc, path, fun, ..., namespaces = namespaces, sessionEncoding = sessionEncoding, addFinalizer = addFinalizer)
```

**Comment by sbpcs59** [comment]
From httr::content() help:

WARNING
When using content() in a package, DO NOT use on as = "parsed". Instead, check the mime-type is what you expect, and then parse yourself. This is safer, as you will fail informatively if the API changes, and you will protect yourself against changes to httr.

Yep, the API changed.

**Comment by sbpcs59** [comment]
A potential solution is to return the http request content as text, and parse it
manually, so from:

```
    xml <- content(response, as="parsed")
```
to
```
 xml <- xmlParse(content(response, as="text"))
```

**Comment by sbpcs59** [comment]
The above fix has been applied where needed, and now all unit tests are passing. Fix applied in commit a57d1f18160fa9a861b7351a93a60d747e6bf95d
